src/test/cypress/support/pageobjects/exercises/quiz/MultipleChoiceQuiz.ts

```diff
@@ -1,23 +1,21 @@
 import { BASE_API, POST } from '../../../constants';
 
 export class MultipleChoiceQuiz {
-    getQuizBody() {
-        return cy.get('.mc-question');
+    getQuizBody(quizQuestionId: string) {
+        return cy.get('#question' + quizQuestionId);
     }
 
-    tickAnswerOption(optionNumber: number) {
-        this.getQuizBody()
+    tickAnswerOption(optionNumber: number, quizQuestionId = '0') {
+        this.getQuizBody(quizQuestionId)
             .get('#answer-option-' + optionNumber)
-            .find('.svg-inline--fa')
+            .find('#mc-answer-selection-' + optionNumber)
             .first()
```

Comment:
Is the `.first()` still needed here? We already query the element with a unique id. The previous selector should only return one element

Reply:
it is not. good point.